package.json

```diff
@@ -73,6 +78,42 @@
 			{
 				"command": "extension.configureLaunchDarkly",
 				"title": "LaunchDarkly: Configure"
+			},
+			{
+				"command": "ldFeatureFlags.refreshEntry",
+				"title": "LaunchDarkly: Refresh Feature Flags",
+				"icon": {
+					"light": "resources/light/refresh.svg",
+					"dark": "resources/dark/refresh.svg"
+				}
+			},
+			{
+				"command": "ldFeatureFlags.copyKey",
+				"title": "LaunchDarkly: Copy Key to Clipboard",
+				"icon": {
+					"light": "resources/light/files.svg",
+					"dark": "resources/dark/files.svg"
+				}
+			},
+			{
+				"command": "launchdarkly.treeviewrefresh",
+				"title": "LD Refresh"
+			},
+			{
+				"command": "ldFeatureFlags.toggleFlag",
+				"title": "LaunchDarkly: Copy Key to Clipboard",
+				"icon": {
+					"light": "resources/light/files.svg",
+					"dark": "resources/dark/files.svg"
+				}
+			},
+			{
+				"command": "ldFeatureFlags.openBrowser",
+				"title": "LD: Open in Browser",
+				"icon": {
+					"light": "resources/light/link_external.svg",
+					"dark": "resources/dark/link_external.svg"
+				}
```

Comment:
![Screen Shot 2020-05-06 at 11 05 11 AM](https://user-images.githubusercontent.com/9063110/81212437-c06ecd80-8f89-11ea-866f-1b5472f3f6e7.png)
It looks like this PR adds a number of new commands visible from the command palette, including a duplicate for "copy key". My preference would be to remove as many external commands and implement buttons / functionality as inline extension behavior